packages/bridge-ui-v2/src/components/Activities/MobileDetailsDialog.svelte

```diff
@@ -60,7 +60,7 @@
           <h4 class="text-secondary-content">{$t('activities.header.explorer')}</h4>
           <a
             href={`${chainUrlMap[Number(selectedItem.srcChainId)].explorerUrl}/tx/${selectedItem.hash}`}
-            target="_blank">
+            target="noopener">
```

Comment:
noopener is not a valid target. noopener is a rel attribute keyword.

[see Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener)

Reply:
Reverted the change.